prometheus_flask_exporter/__init__.py

```diff
@@ -413,24 +421,32 @@ def teardown_request(exception=None):
 
             response = make_response('Exception: %s' % exception, 500)
 
+            if callable(duration_group):
+                group = duration_group(request)
+            else:
+                group = getattr(request, duration_group)
+
+            request_exceptions_labels = {
+                'method': request.method,
+                'status': 500,
+                'path': request.path,
+            }
+            request_exceptions_labels.update(labels.values_for(response))
+            request_exceptions_metric.labels(**request_exceptions_labels).inc()
+
             if hasattr(request, 'prom_start_time'):
                 total_time = max(default_timer() - request.prom_start_time, 0)
 
-                if callable(duration_group):
-                    group = duration_group(request)
-                else:
-                    group = getattr(request, duration_group)
-
-                histogram_labels = {
+                request_duration_labels = {
```

Comment:
Was this supposed to be this metric? It has hardcoded `500` for status?

Reply:
Oh crap - sorry, that was a copy-pasta error, same below

Reply:
No worries! Now that you found a better way to find the status code, should we use it for this one too instead of the hardcoded 500?